user-guide/Advanced_Functionality/Security/About_DMS_Security/Cloud_connectivity_and_security.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
The content of this file and of https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Cloud_connection_security.html really belongs together. Can you combine these in one file and remove the other file, adding a "see also" link in the appropriate location instead?